extensions/cratecast/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "cratecast",
+  "title": "Search crates.io",
```

Comment:
**suggestion**: crates.io Search
See https://developers.raycast.com/basics/prepare-an-extension-for-store